affected-paths/app/src/main/kotlin/com/squareup/affected/paths/app/utils/CoreOptionsUtils.kt

```diff
@@ -31,6 +31,8 @@ internal fun BaseConfigurationOptions.toCoreOptions(): CoreOptions {
     maxGradleMemory = maxGradleMemory,
     customJvmFlags = listOf("-XX:-MaxFDLimit"),
     customGradleFlags = listOf("--stacktrace"),
-    autoInjectPlugin = autoInject
+    autoInjectPlugin = autoInject,
+    changedFiles = changedFiles,
+    useCustomGradleInstallation = gradleInstallationPath,
```

Comment:
Name `useCustomGradleInstallation` suggests that it would be a boolean, but actually it's a `Path?` - how about just re-using `gradleInstallationPath` name, or `customGradleInstallationPathToUse`.